packages/web/components/Landing/LandingHeader.tsx

```diff
@@ -1,23 +1,114 @@
+/* eslint-disable no-nested-ternary */
 import {
   Box,
   BoxedNextImage,
   Button,
   Flex,
   HStack,
+  Icon,
+  IconButton,
   Link,
   Stack,
+  Text,
+  Tooltip,
+  usePrefersReducedMotion,
+  useToast,
   VStack,
 } from '@metafam/ds';
 import OctoBg from 'assets/baby_octo.png';
 import MetaGameLogo from 'assets/logo.png';
 import { MetaLink } from 'components/Link';
 import { SetStateAction } from 'jotai';
-import { Dispatch, ReactNode, useState } from 'react';
+import { get, set } from 'lib/store';
+import { Dispatch, ReactNode, useCallback, useEffect, useState } from 'react';
+import { FaToggleOff, FaToggleOn } from 'react-icons/fa';
+import { MdClose } from 'react-icons/md';
 
 import { AnimatedWaves, upDownAnimation } from './animations';
+import { LandingConnectButton } from './LandingConnectButton';
 
 export const LandingHeader: React.FC = () => {
   const [toggle, setToggle] = useState(false);
+  const prefersReducedMotion = usePrefersReducedMotion();
+  const savedMotionPreference = get('MotionPreference');
+  const [noMotion, setNoMotion] = useState(
+    savedMotionPreference === 'off'
+      ? true
+      : savedMotionPreference === 'on'
+      ? false
+      : prefersReducedMotion,
```

Comment:
Elsewhere I had commented that we store the `no-motion` in localStorage but looks like you're already doing it! 🥳

Lets `setNoMotion` here in a useEffect. This `useState` will be run on the server during static build and won't be set correctly. (This may explain the TODO you've added below)

Reply:
> Elsewhere I had commented that we store the `no-motion` in localStorage but looks like you're already doing it! partying_face

😄 I replied to that and now see you've seen that I handled it. 🤣 

> Lets `setNoMotion` here in a useEffect. This `useState` will be run on the server during static build and won't be set correctly. (This may explain the TODO you've added below)

Ah, thanks. I'll address that